internals/templates/app.js

```diff
@@ -83,7 +83,10 @@ if (module.hot) {
 
 // Chunked polyfill for browsers without Intl support
 if (!window.Intl) {
-  System.import('intl').then(() => render(translationMessages));
+  Promise.all([
+    System.import('intl'),
+    System.import('intl/locale-data/jsonp/en.js'),
```

Comment:
I'm not sure but I think there is race condition in here.
in case, `System.import('intl/locale-data/jsonp/en.js'),` is run before `System.import('intl'),` it will cause this bug in Safari

![image](https://cloud.githubusercontent.com/assets/2979072/17277706/994f4e20-5773-11e6-8394-6fde5c5a5117.png)